packages/table/test/rowHeaderTests.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
```

Comment:
the license has been updated. can you please merge master, delete this comment block, and run the lint auto-fixer to get the new file header?